test/HeapPageIdTest.cpp

```diff
@@ -0,0 +1,83 @@
+#include <gtest/gtest.h>
+#include "heap_page_id.h"
+
+namespace emerald {
+class PidTests {
+public:
```

Comment:
Please add one space before `public` and `private`. They are supposed to be like "half-indented".